### PR TITLE
[CIVIS-9838] Add docs for AWS CodePipeline log correlation

### PR DESCRIPTION
**Pull request by ManuelPalenzuelaDD**
### What does this PR do? What is the motivation?

Add docs for  the new AWS CodePipeline log correlation feature.

### Merge instructions

- [x] Please merge after reviewing

### CODE REVIEW

File: content/en/continuous_integration/pipelines/awscodepipeline.md [hunk 1]
```diff
@@ -30,9 +30,11 @@ Set up tracing on AWS CodePipeline to collect data about pipeline executions, an
 | Pipeline Visibility | Platform | Definition |
 |---|---|---|
 | [Partial retries][14] | Partial pipelines | View partially retried pipeline executions. |
-| [Running pipelines][15] | Running pipelines | View pipeline executions that are running. |
+| *[Running pipelines][15] | Running pipelines | View pipeline executions that are running. |
+| **Logs correlation | Logs correlation	| Correlate pipeline and job spans to logs and enable [job log correlation][16]. |
 
-**Note**: AWS CodePipeline running pipelines don't have Git information until they have finished.
+*AWS CodePipeline running pipelines don't have Git information until they have finished.\
+**AWS CodePipeline logs correlation is currently only available for AWS CodeBuild actions.
```
Comment (buraizu) [inline content/en/continuous_integration/pipelines/awscodepipeline.md:37]:
```suggestion
**AWS CodePipeline logs correlation is only available for AWS CodeBuild actions.
```

File: content/en/continuous_integration/pipelines/awscodepipeline.md [hunk 2]
```diff
@@ -111,6 +113,10 @@ Name the variable `DD_PIPELINE_EXECUTION_ID`, and the value `#{codepipeline.Pipe
 
 The steps above allow you to add the pipeline execution ID to your CodeBuild action environment variables. For more information on working with variables, see the [official AWS guide][10].
 
+### Enable log correlation
+
+The AWS CodePipeline integration supports correlating **CodeBuild** job's actions with their respective job and pipeline spans. To enable log collection for your CodeBuild actions, follow the steps in the general [AWS logs forwarding guide][17].
```
Comment (buraizu) [inline content/en/continuous_integration/pipelines/awscodepipeline.md:118]:
```suggestion
The AWS CodePipeline integration supports correlating **CodeBuild** job actions with their respective job and pipeline spans. To enable log collection for your CodeBuild actions, follow the steps in the [AWS log forwarding guide][16].
```

File: content/en/continuous_integration/pipelines/awscodepipeline.md [hunk 3]
```diff
@@ -135,4 +141,6 @@ View your data on the [Pipelines][11] and [Pipeline Executions][12] pages after
 [12]: https://app.datadoghq.com/ci/pipeline-executions
 [13]: https://docs.datadoghq.com/continuous_integration/troubleshooting/#the-default-branch-is-not-correct
 [14]: /glossary/#partial-retry
-[15]: /glossary/#running-pipeline
\ No newline at end of file
+[15]: /glossary/#running-pipeline
+[16]: /continuous_integration/pipelines/awscodepipeline/#enable-log-correlation
+[17]: /logs/guide/send-aws-services-logs-with-the-datadog-lambda-function
```
Comment (buraizu) [inline content/en/continuous_integration/pipelines/awscodepipeline.md:146]:
```suggestion
[16]: /logs/guide/send-aws-services-logs-with-the-datadog-lambda-function
```

File: content/en/continuous_integration/pipelines/awscodepipeline.md [hunk 4]
```diff
@@ -30,9 +30,11 @@ Set up tracing on AWS CodePipeline to collect data about pipeline executions, an
 | Pipeline Visibility | Platform | Definition |
 |---|---|---|
 | [Partial retries][14] | Partial pipelines | View partially retried pipeline executions. |
-| [Running pipelines][15] | Running pipelines | View pipeline executions that are running. |
+| *[Running pipelines][15] | Running pipelines | View pipeline executions that are running. |
+| **Logs correlation | Logs correlation	| Correlate pipeline and job spans to logs and enable [job log correlation][16]. |
```
Comment (buraizu) [inline content/en/continuous_integration/pipelines/awscodepipeline.md:34]:
```suggestion
| **Logs correlation | Logs correlation	| [Enable job log correlation](#enable-log-correlation) to correlate pipeline and job spans to logs. |
```

**Review by buraizu — APPROVED** [review]
Approving with some minor edits suggested.

File: content/en/continuous_integration/pipelines/awscodepipeline.md (hunk 4 above)
Comment (ManuelPalenzuelaDD) [reply content/en/continuous_integration/pipelines/awscodepipeline.md]: We use the `Correlate pipeline and job spans to logs and enable job log correlation` in all the docs for other providers - imho we should be consistent.

File: content/en/continuous_integration/pipelines/awscodepipeline.md [hunk 5]
```diff
@@ -111,6 +113,12 @@ Name the variable `DD_PIPELINE_EXECUTION_ID`, and the value `#{codepipeline.Pipe
 
 The steps above allow you to add the pipeline execution ID to your CodeBuild action environment variables. For more information on working with variables, see the [official AWS guide][10].
 
+### Enable log correlation
+
+The AWS CodePipeline integration supports correlating **CodeBuild** actions with their respective job and pipeline spans. To enable log collection for your CodeBuild actions, follow the steps in the [AWS log forwarding guide][16].
```
Comment (buraizu) [inline content/en/continuous_integration/pipelines/awscodepipeline.md:118]:
```suggestion
The AWS CodePipeline integration supports correlating **CodeBuild** actions with their respective job and pipeline spans. To enable log collection for your CodeBuild actions, see [Enable logging for your AWS service][16] in the AWS log forwarding guide.
```

**Review by buraizu — CHANGES_REQUESTED** [review]
Some additional minor suggestions per the [style guide](https://github.com/DataDog/documentation/blob/master/CONTRIBUTING.md#tense)

**Comment by ManuelPalenzuelaDD** [comment]
@buraizu Thanks for the suggestions! I've chosen to edit one of them (see latest commit) as the AWS CodeBuild service isn't in [the table](https://docs-staging.datadoghq.com/manuel.palenzuela/CIVIS-9838-aws-code-pipeline-logs-update-documentation/logs/guide/send-aws-services-logs-with-the-datadog-lambda-function/?tab=awsconsole#enable-logging-for-your-aws-service).

**Review by buraizu — APPROVED** [review]

**Pull request merged**